01_fib/eastrd/main_test.go

```diff
@@ -0,0 +1,116 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"strconv"
+	"testing"
+)
+
+func TestFibOutput1(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	fib(7)
+
+	expected := strconv.Quote("1\n1\n2\n3\n5\n8\n13\n")
+	actual := strconv.Quote(buf.String())
+
+	if expected != actual {
+		t.Errorf("Unexpected output in main()")
+	}
+}
+
+func TestFibOutput2(t *testing.T) {
```

Comment:
Look into table driven tests to avoid the reputation here:
- [our sample from slides](https://github.com/anz-bank/go-slides/blob/master/samples/testing/basic/cases_test.go)
- [Dave Cheney post](https://dave.cheney.net/2019/05/07/prefer-table-driven-tests)